examples/python/gpio_advanced.py

```diff
@@ -21,34 +21,39 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE
+#
+# Example Usage: Reads GPIO 6 through interrupt
```

Comment:
This description is IMHO a bit misleading. The idea of this functionality and the example is not in the specific fact we *read* the GPIO when the interrupt happens, but in the general fact is that we can *execute a predefined piece of code upon GPIO state change*, so I'd suggest highlighting namely that.

Reply:
Okay. How about `Trigger ISR upon GPIO state change`?

Reply:
Yep, sounds good.